.github/workflows/CI.yml

```diff
@@ -84,6 +84,18 @@ jobs:
       run: |
         CHPL_HOME=$PWD make run-dyno-linters
 
+  make_check_llvm_none:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/${{ github.repository_owner }}/chapel-github-ci:latest
+      credentials:
+        username: ${{ github.actor }}
+        password: ${{ secrets.github_token }}
+    steps:
+    - uses: actions/checkout@v3
+    - name: make check_llvm_none
```

Comment:
Not sure how important it is, but I think this field is the name that appears when you look at job details. Looking at other similar jobs, it looks like this should maybe refer to something real. i.e. I can do `make check` on command line, there is no `make check_llvm_none`, however. If we can have two actions with the same name, this should maybe just be `make check` because that's what smokeTest ends up doing.

I am way out of my comfort zone, though.

Reply:
I changed it to set llvm_none make check